src/utils/data_types.h

```diff
@@ -14,7 +14,6 @@
 #include <cassert>
 
 #include "global_params.h"
-#include "openmvg_descriptor.hpp"
```

Comment:
Yes.
I prepare a PR with more features.
Perhaps it's better to wait the update PR.
You can still review the other changes.